src/main/java/org/embulk/output/bigquery_java/converter/BigqueryStringConverter.java

```diff
@@ -106,6 +108,11 @@ public static void convertAndSet(ObjectNode node, String name, String src, Bigqu
                     }
                 }
                 break;
+            case NUMERIC:
+                // BigQuery NUMERIC型のスケール最大値が9なのでデフォルト値を9にする
```

Comment:
English please

Reply:
got it !

Reply:
Fixed https://github.com/trocco-io/embulk-output-bigquery_java/pull/93/commits/237377bc04f71c1f6edb83ebd639555d8b2ba895